src/components/LoanCards/Buttons/LendIncrementButton.vue

```diff
@@ -77,8 +81,17 @@ export default {
 			// IF we wanted to show this interface on loans with less than 25 remaining they would see the selector
 			const minAmount = parseFloat(this.amountLeft < 25 ? this.loan.minNoteSize : 25); // 25_hard_coded
 			// cap at 20 prices
-			return buildPriceArray(this.amountLeft, minAmount).slice(0, 20);
-		}
+			const priceArray = buildPriceArray(parseFloat(this.amountLeft), minAmount).slice(0, 20);
+			// eslint-disable-next-line
+			if (this.isCompleteLoanActive && !priceArray.includes(Number(this.amountLeft).toFixed())) {
+				priceArray.push(Number(this.amountLeft).toFixed());
```

Comment:
`Number(this.amountLeft).toFixed()` can be stored in a const above the if and used twice to make this logic a little clearer.

Reply:
added!